UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/RayPerception.cs

```diff
@@ -3,17 +3,11 @@
 
 public abstract class RayPerception : MonoBehaviour
```

Comment:
does this need to inherit from MonoBehaviour?

Reply:
never mind, naming got to me.  Can you change to something like `RayPerceptionComponent`?

Reply:
This is the old code. I could add `[Obsolete]` if we want to make it more obvious users should switch to the new system.

Reply:
I think that would be a good idea.

Reply:
Cool, added `[Obsolote]`